vue/components/ui/atoms/dropdown/dropdown.vue

```diff
@@ -143,11 +143,17 @@
     width: auto;
 }
 
-.dropdown-container .dropdown > .dropdown-item:hover,
-.dropdown-container .dropdown > .dropdown-item:active,
-.dropdown-container .dropdown > .dropdown-item.selected,
-.dropdown-container .dropdown > .dropdown-item.highlighted {
+.dropdown-container .dropdown > .dropdown-item:not(.disabled):hover,
+.dropdown-container .dropdown > .dropdown-item:not(.disabled):active,
+.dropdown-container .dropdown > .dropdown-item:not(.disabled).selected,

```

Comment:
the selected should not be with the not disabled part, since I would say it might be possible to have an item selected by default and disabled. What do you think?

Reply:
@joamag The selected was not using the negative selector due to this possibility

Reply:
Yes, I got it but it's weird to have a selected disabled element... but gain I'm not sure what's defined to be a selected item. I'm going to change that in master then